common/utils/EncryptionManager.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const crypto = require('crypto');
+const config = require('../config');
+const CONST = require('../constants');
+
+class EncryptionManager {
+
+  /**
+   * @param text test to be encrypted
+   */
+  encrypt(text) {
+    const cipher = crypto.createCipher(CONST.APISERVER.ENCRYPTION.AES_256_ALGORITHM, config.apiserver.encryption.password);
+    let crypted = cipher.update(text, CONST.APISERVER.ENCRYPTION.INPUT_ENCODING, CONST.APISERVER.ENCRYPTION.OUTPUT_ENCODING);
```

Comment:
Though not a big problem but note that this is a synchronous operation which blocks the event loop.

Reply:
anything you would want to change?

Reply:
No, let's keep it like this for now. There are asynchronous versions via C++ node modules available but that might add some build-level complexity.